WcaOnRails/config/initializers/wicked_pdf.rb

```diff
@@ -13,9 +13,8 @@
 WickedPdf.config = {
   # Path to the wkhtmltopdf executable: This usually isn't needed if using
   # one of the wkhtmltopdf-binary family of gems.
-  exe_path: '/usr/local/bin/wkhtmltopdf',
```

Comment:
Super nitpick: Comment this out instead of deleting it to mimick how the configuration file looked before your change